src/main/java/com/cgdpd/library/controller/BookController.java

```diff
@@ -35,4 +39,12 @@ public CreateBookResponseDTO createBook(@RequestBody CreateBookRequestDTO reques
     public DetailedBookDTO getBook(@PathVariable Isbn13 isbn13) {
         return bookService.getDetailedBookByIsbn13(isbn13);
     }
+
+    @GetMapping("/book")
```

Comment:
This mapping doesn't look good, it ends up as `/books/book?params=`
Leave it empty